packages/core/contracts/financial-templates/expiring-multiparty/ExpiringMultiPartyCreator.sol

```diff
@@ -93,8 +93,17 @@ contract ExpiringMultiPartyCreator is ContractCreator, Testable, Lockable {
         require(bytes(params.syntheticSymbol).length != 0, "Missing synthetic symbol");
         require(params.withdrawalLiveness != 0, "Withdrawal liveness cannot be 0");
         require(params.liquidationLiveness != 0, "Liquidation liveness cannot be 0");
+        require(params.expirationTimestamp > now, "Invalid expiration time");
         _requireWhitelistedCollateral(params.collateralAddress);
 
+        // We don't want EMP deployers to be able to intentionally or unintentionally set
+        // liveness periods that could induce arithmetic overflow, but we also don't want
+        // to be opinionated about what livenesses are "correct", so we will somewhat
+        // arbitrarily set the livness upper bound at 1 year. In practice, liveness
+        // periods equal to 1 year would make the respective EMP unusable.
+        require(params.withdrawalLiveness < 31536000, "Withdrawal liveness too large");
+        require(params.liquidationLiveness < 31536000, "Liquidation liveness too large");
```

Comment:
I thought we wanted to go with 100 years?

nit: I think you can specify this in a slightly more readable way:
```
require(params.withdrawalLiveness < 100 years, "Withdrawal liveness too large");
```

See https://solidity.readthedocs.io/en/v0.4.24/units-and-global-variables.html#time-units for info on these built-in time converters.

Reply:
I changed to 5200 weeks because `years` key word is deprecated due to leap year ambiguity